packages/login/src/vaadin-login-form.js

```diff
@@ -110,7 +109,7 @@ class LoginForm extends LoginMixin(ElementMixin(ThemableMixin(PolymerElement)))
   /** @protected */
   connectedCallback() {
     super.connectedCallback();
-    this._handleInputKeydown = this._handleInputKeydown.bind(this);
```

Comment:
No need to bind this listener, because it's used in a template and Polymer handles that.